server/index.js

```diff
@@ -7,6 +7,9 @@ const img = require("./database/img.json");
 const database = require("./database/database");
 const style = require("./database/style/style");
```

Comment:
<h3>the new footer CSS </h3>
<h5>developed by figma</h5>

```JSON
{
"container":"display:flex; width: 1228px; padding:padding: 80px 0px 40px 0px; align-items:center;",
"form": "display:flex; flex-direction:column; align-items:center; align-self: stretch;",
"div.text-title":"display: flex; padding-right:8px; align-items:flex-start;", 
"p.text-title": "color: #FFF; font-family: Inter; font-size: 26px; font-style: normal; font-weight: 700; line-height: 39px;  letter-spacing: 2px; text-transform: uppercase; "
"div.text-description":"display:flex; padding-top:24px; justify-content:center; align-items:flex-start;",
"p.text-description": "color: #000; text-align: center; font-family: Inter; font-size: 16px; font-style: normal; font-weight: 400; line-height: 24px;",
"div.stack-margin" : "display: flex; padding-top: 24px; flex-direction: column; align-items: center; align-self: stretch;",
"div.stack":"display: flex; padding: 0px 360px; flex-direction: column; align-items: center; gap: 7px; align-self: stretch;",
"div.container": "display: flex; max-width: 450px; justify-content: center; align-items: center; align-self: stretch;",
"div.link":"width: 470.66px; height: 52px;" ,
"separator": "width: 2px; height:20px; flex-srhink: 0;",
"p.link-text": "display: inline-flex; padding-bottom: 0px; align-items: flex-start; color: #000; font-family: Inter; font-size: 14px; font-style: normal; font-weight: 500; line-height: 22.5px;", 
"div.dev-employ": "widht: 150px; height:25.30px; flex-shrink: 0; ",
"dev-employ.svg" {
"css": "width: 150px; height:25px; flex-shrink:0;", 
"svg": [
"<svg xmlns="http://www.w3.org/2000/svg" width="151" height="26" viewBox="0 0 151 26" fill="none">
  <g clip-path="url(#clip0_5_844)">
    <mask id="mask0_5_844" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="26" height="26">
      <path d="M13.0669 25.7616C20.0705 25.7616 25.7481 20.0841 25.7481 13.0805C25.7481 6.07684 20.0705 0.399292 13.0669 0.399292C6.06329 0.399292 0.385742 6.07684 0.385742 13.0805C0.385742 20.0841 6.06329 25.7616 13.0669 25.7616Z" fill="url(#paint0_linear_5_844)"/>
    </mask>
    <g mask="url(#mask0_5_844)">
      <path d="M13.0669 25.3226C19.828 25.3226 25.3091 19.8416 25.3091 13.0804C25.3091 6.31926 19.828 0.838257 13.0669 0.838257C6.30571 0.838257 0.824707 6.31926 0.824707 13.0804C0.824707 19.8416 6.30571 25.3226 13.0669 25.3226Z" fill="url(#paint1_linear_5_844)" stroke="url(#paint2_linear_5_844)" stroke-width="0.877928"/>
      <path d="M18.8803 9.6093C17.9054 7.34479 15.6704 5.7644 13.0668 5.7644C10.4631 5.7644 8.22813 7.34479 7.25323 9.6093C6.60333 9.66532 6.09212 10.2135 6.09212 10.8856V13.4463C6.09212 14.1184 6.60333 14.6665 7.25323 14.7226C8.22813 16.9871 10.4631 18.5675 13.0668 18.5675C15.6704 18.5675 17.9054 16.9871 18.8803 14.7226C19.5302 14.6665 20.0414 14.1184 20.0414 13.4463V10.8856C20.0414 10.2135 19.5302 9.66532 18.8803 9.6093ZM17.5052 11.5258V12.166C17.5052 14.2865 15.8011 16.0069 13.7008 16.0069H12.4327C10.3324 16.0069 8.62835 14.2865 8.62835 12.166V11.5258C8.62835 10.4655 9.48038 9.60532 10.5305 9.60532H15.603C16.6531 9.60532 17.5052 10.4655 17.5052 11.5258ZM11.6916 12.2739L11.4539 11.4258C11.4182 11.2937 11.2993 11.2058 11.1646 11.2058C11.0299 11.2058 10.911 11.2937 10.8753 11.4258L10.6376 12.2739L9.79741 12.514C9.66661 12.55 9.57943 12.6701 9.57943 12.8061C9.57943 12.9421 9.66661 13.0622 9.79741 13.0981L10.6376 13.3382L10.8753 14.1864C10.911 14.3185 11.0299 14.4065 11.1646 14.4065C11.2993 14.4065 11.4182 14.3185 11.4539 14.1864L11.6916 13.3382L12.5318 13.0981C12.6625 13.0622 12.7497 12.9421 12.7497 12.8061C12.7497 12.6701 12.6625 12.55 12.5318 12.514L11.6916 12.2739ZM8.6561 18.4275C6.0406 19.4717 4.18994 22.0484 4.18994 25.061C4.18994 25.7172 4.717 26.2494 5.36691 26.2494H9.26241V23.6887C9.26241 22.9805 9.82907 22.4084 10.5305 22.4084H15.603C16.3044 22.4084 16.8711 22.9805 16.8711 23.6887V26.2494H20.7666C21.4165 26.2494 21.9436 25.7172 21.9436 25.061C21.9436 22.0484 20.0929 19.4717 17.4774 18.4275C16.2331 19.3197 14.7113 19.8478 13.0668 19.8478C11.4222 19.8478 9.90045 19.3197 8.6561 18.4275ZM11.1646 23.6887C10.8158 23.6887 10.5305 23.9768 10.5305 24.3289V26.2494H11.7986V24.3289C11.7986 23.9768 11.5133 23.6887 11.1646 23.6887ZM14.9689 24.969C15.3177 24.969 15.603 24.681 15.603 24.3289C15.603 23.9768 15.3177 23.6887 14.9689 23.6887C14.6202 23.6887 14.3349 23.9768 14.3349 24.3289C14.3349 24.681 14.6202 24.969 14.9689 24.969Z" fill="url(#paint3_linear_5_844)"/>
    </g>
  </g>
  <defs>
    <linearGradient id="paint0_linear_5_844" x1="13.0669" y1="0.399292" x2="13.0669" y2="25.7616" gradientUnits="userSpaceOnUse">
      <stop stop-color="#477EEB"/>
      <stop offset="1" stop-color="#0F388A"/>
    </linearGradient>
    <linearGradient id="paint1_linear_5_844" x1="13.0669" y1="0.399271" x2="13.0669" y2="25.7616" gradientUnits="userSpaceOnUse">
      <stop stop-color="#477EEB"/>
      <stop offset="1" stop-color="#0F388A"/>
    </linearGradient>
    <linearGradient id="paint2_linear_5_844" x1="13.0669" y1="0.399271" x2="13.0669" y2="25.7616" gradientUnits="userSpaceOnUse">
      <stop stop-color="white"/>
      <stop offset="1" stop-color="#CCCCCC"/>
    </linearGradient>
    <linearGradient id="paint3_linear_5_844" x1="13.0668" y1="5.7644" x2="13.0668" y2="26.2494" gradientUnits="userSpaceOnUse">
      <stop/>
      <stop offset="0.9999" stop-color="#CCCCCC"/>
      <stop offset="1" stop-color="white" stop-opacity="0"/>
    </linearGradient>
    <clipPath id="clip0_5_844">
      <rect width="150" height="25.3623" fill="white" transform="translate(0.385742 0.399292)"/>
    </clipPath>
  </defs>
</svg>"
] 
}, 
"p.fullstack-exercises":"color: #000; font-family: Poppins; font-size: 12px; font-style: normal; font-weight: 500;line-height: 20px;  line-height: 20px;",
"p.footer-text": "display:flex; padding-bottom: 0px; align-items: flex-start; gap: 2.22px;"
}
```

<h3> The element </h3>

![image](https://github.com/97revenge/exercises-fullstack/assets/80254945/0a4146c5-2fd1-438f-8ebb-a703c1c8a12e)